tests/models/vilt/test_modeling_vilt.py

```diff
@@ -227,11 +227,7 @@ class ViltModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
         if is_torch_available()
         else ()
     )
-    pipeline_model_mapping = (
-        {"feature-extraction": ViltModel, "visual-question-answering": ViltForQuestionAnswering}
-        if is_torch_available()
-        else {}
-    )
+    pipeline_model_mapping = {"visual-question-answering": ViltForQuestionAnswering} if is_torch_available() else {}
```

Comment:
nit: just wondering why `"feature-extraction": ViltModel` is not kept (like in other places)